agent/src/main.rs

```diff
@@ -400,15 +404,17 @@ where
      */
     let expected_downstairs_instances = regions
         .iter()
-        .filter(|r| r.state == State::Created)
+        .filter(|r| r.state == State::Created || r.state == State::Requested)
```

Comment:
I don't think this is right - if I remember correctly this would start a downstairs process for a region that either hasn't had its dataset created yet, or could still be cloning.